pkg/generate/app/cmd/newapp.go

```diff
@@ -542,8 +543,36 @@ func (c *AppConfig) buildPipelines(components app.ComponentReferences, environme
 						}
 					}
 				}
-				if pipeline, err = app.NewBuildPipeline(ref.Input().String(), input, c.OutputDocker, strategy, c.GetBuildEnvironment(environment), source); err != nil {
-					return nil, fmt.Errorf("can't build %q: %v", ref.Input(), err)
+
```

Comment:
is it the same for --name? What kind of use of new-app /new-build produces 2 builds?